api-docs/leads.md

```diff
@@ -0,0 +1,332 @@
+  "zip": "84115",
+  "age": 40,
+  "gender": "female",
+  "intakeServices": [
+    {
+      "id": 1,
+      "serviceName": "Citizenship",
+      "serviceDescription": "Gain United States citizenship"
+    }
+  ],
+  "isDeleted": false,
+  "createdBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  },
+  "lastUpdatedBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  }
+},
+{
+  "id": 3,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "pending",
+  "firstName": "Ron",
+  "lastName": "Weasley",
+  "fullName": "Ron Weasley",
+  "phone": "555-555-5555",
+  "smsConsent": true,
+  "zip": "84115",
+  "age": 39,
+  "gender": "male",
+  "intakeServices": [
+    {
+      "id": 1,
+      "serviceName": "Citizenship",
+      "serviceDescription": "Gain United States citizenship"
+    }
+  ],
+  "isDeleted": false,
+  "createdBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  },
+  "lastUpdatedBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  }
+})
+```
+
+## Create a single Lead
+
+### Request
+
+```http
+POST /api/leads
+```
+
+```json
+{
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "pending",
+  "firstName": "Joel",
+  "lastName": "Denning",
+  "fullName": "Ron Weasley",
+  "phone": "555-555-5555",
+  "smsConsent": false,
+  "zip": "84115",
+  "age": 25,
+  "gender": "male",
+  "intakeServices": [1, 5, 8]
+}
+```
+
+**_Notes_**
+
+- `gender` is an enum with possible values of `female`, `male`, `transgender`, `nonbinary`, `other`.
+- `intakeServices` is an array of integers service ids. See (/api-docs/list-services.md).
+
+### Response
+
+#### Success
+
+The response object will be the same as if you do a `GET /api/leads/:id`
+
+```json
+{
+  "id": 1,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "pending",
+  "firstName": "Joel",
+  "lastName": "Denning",
+  "fullName": "Joel Denning",
+  "phone": "555-555-5555",
+  "smsConsent": true,
+  "zip": "84115",
+  "age": 25,
+  "gender": "male",
+  "intakeServices": [
+    {
+      "id": 1,
+      "serviceName": "Citizenship",
+      "serviceDescription": "Gain United States citizenship"
+    }
+  ],
+  "isDeleted": false,
+  "createdBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  },
+  "lastUpdatedBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  }
+}
+```
+
+**Notes**
+
+- `created.timestamp` and `lastUpdated.timestamp` are unix timestamps
```

Comment:
unix timestamps are numbers, not strings. They are always UTC. We do not use them. The format we use is the ISO date time format.

Reply:
Should it be changed in the create-client.md too then?